src/Tribe/RSVP.php

```diff
@@ -466,6 +458,15 @@ public function generate_tickets( ) {
 
 			$has_tickets = true;
 
+			/**
+			* RSVP specific action fired just before a RSVP-driven attendee ticket for an event is generated
+			*
+			* @param $event_id ID of event
+			* @param $ticket Ticket of product
+			* @param $_POST post paremeters comes from RSVP Form
+			*/
+			do_action( 'tribe_tickets_rsvp_before_tickets_generated', $event_id, $ticket, $_POST );
```

Comment:
Maybe I'm grinding a personal axe here, but lots of our ticket code is confusing because of our loose use of terms like tickets. For instance, here we're passing `$ticket` to a callback that fires before tickets are generated. Might `$ticket_type` help to differentiate between the two types of 'ticket'?

Optional, might just be the way I read things.